Libraries/LinkingIOS/RCTLinkingManager.m

```diff
@@ -47,7 +47,20 @@ + (BOOL)application:(UIApplication *)application
   sourceApplication:(NSString *)sourceApplication
          annotation:(id)annotation
 {
-  NSDictionary<NSString *, id> *payload = @{@"url": URL.absoluteString};
+  NSURLComponents *urlComponents = [NSURLComponents componentsWithURL:URL
+                                              resolvingAgainstBaseURL:NO];
+  NSMutableDictionary<NSString *, NSString *> *parameters = [NSMutableDictionary new];
+  for (NSURLQueryItem *queryItem in urlComponents.queryItems) {
```

Comment:
NSURLComponents.queryItems is an iOS8+ api, and we need to support iOS7+ (We got burned by this once already :-))